### ISSUE TITLE
Pymol won't install plugin

### ISSUE THREAD

**Issue by sfhbarnett** [APAJanssen/Alphafold2import#2]
Hi,

I'm running into an issue installing the updated version of the plugin from two days ago. Whenever I try to install it I get this message: `Plugin "Alphafold2import-main" has been installed but initialization failed.` and the `fetchAF2` command is not defined when run. The previous version works however. I have tried uninstalling and reinstalling both and the behaviour is consistent.

Running on a MacOS Monterey
PyMOL(TM) Molecular Graphics System, Version 2.5.0

Thanks,
Sam

**Comment by APAJanssen** [comment]
Hi Sam,
Unfortunately I cannot reproduce the issue in either Windows 10 or Ubuntu 20.04, here both work just as expected.
Some suggestions:
- I've attached a new .zip here, perhaps this does (somehow) work? [Alphafold2import.zip](https://github.com/APAJanssen/Alphafold2import/files/8231147/Alphafold2import.zip)
- Are you running PyMOL from Schrödinger or the open source variant? 
- You could try launching PyMOL from the terminal and paste the (usually more extensive) error output that shows to further track down the issue.

**Comment by sfhbarnett** [comment]
Thanks, I tried with the zip and got the same result. This is the open source version of PyMOL and the terminal output gives this and definitely hints at the problem. 

    No module named 'requests'
    Unable to initialize plugin 'Alphafold2import-main' (pmg_tk.startup.Alphafold2import-main).

I installed `requests` through terminal in the same environment in which I run terminal and this did not seem to solve the issue, PyMOL still thinks requests is missing even though I verified it's present in the environment with `pip list` (gives requests = 2.27.1).

I then tried installing requests directly in PyMOL with the following code and that seemed to fix it! not sure why PyMOL was having issues with the module before.

    import subprocess 
    import sys
    subprocess.check_call([sys.executable, "-m", "pip", "install","requests"])
